### PR TITLE
Fix Issue 5700 : Added 'BCC to Salesforce' in Email Instructors

### LINKED ISSUE #5700: 'Email Instructors' feature should have 'BCC to Salesforce' option just like Ticket replies
We recently added an option to email instructors from the course page. It would be very useful to also add 'BCC to Salesforce' to this by default, as that will help us keep track of communication we do with a given instructor more easily.

(It should work the same way as in the Ticket system, where 'BBC to Salesforce' is checked by default in the UI but can be unchecked to disable it for an individual message.)

### PR DESCRIPTION
**Pull request by prathamVaidya**
## What this PR does

Fixes #5700 'Email Instructors' feature should have 'BCC to Salesforce' which is true by default.

- [x] Added BCC To salesforce option.
- [x] Added rspec test for `#notify_instructors` controller function.

## Screenshots

image

### CODE REVIEW

File: app/assets/javascripts/components/overview/notify_instructors_button.jsx [hunk 1]
```diff
@@ -50,16 +51,30 @@ const NotifyInstructorsButton = (props) => {
...
-            onChange={(_, value) => {
-              dispatch({ type: ALERT_INSTRUCTOR_UPDATE_MESSAGE, payload: value });
-            }}
-            value_key="notification_message"
-            editable
-            placeholder={I18n.t('course_instructor_notification.write_message_placeholder')}
-          />
+          <div className="form-group">
+            <TextAreaInput
+              id={'notification-message'}
```
Comment (AmitJoki) [inline app/assets/javascripts/components/overview/notify_instructors_button.jsx:56]: No need to use `{}` for literals. `id='notification-message'` would suffice.

File: app/mailers/instructor_notification_mailer.rb [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
     @alert = alert
     set_email_parameters
     params = { to: @instructors.pluck(:email),
-               subject: @alert.subject }
+               subject: @alert.subject,
+               bcc: [
+                 @alert.sender_email,
+                 bcc_to_salesforce ? @alert.bcc_to_salesforce_email : nil
+               ].compact }
+    p params
```
Comment (ragesoss) [inline app/mailers/instructor_notification_mailer.rb:18]: Leftover debugging?

Comment (prathamVaidya) [reply app/mailers/instructor_notification_mailer.rb:18]: Oops!

**Comment by prathamVaidya** [comment]
workflow failure looks unrelated

**Comment by Formasitchijoh** [comment]
@ragesoss  can i continue with this task

**Pull request merged**